sparx/templates/wikiparser.py

```diff
@@ -236,6 +236,15 @@ def construct_keyword_dict():
 	#keyword_dict["--symmetry"]                    = SXkeyword_map(2, "sym")                 # --symmetry
 	#keyword_dict["--max_occupy"]                  = SXkeyword_map(2, "int")                 # --max_occupy
 
+	# Added keywords for ctf refine
```

Comment:
Are this keyword_dict entries necessary to build the GUI?

Reply:
And the others below

Reply:
Yes, they are necessary. The other keywords are defined by other tools before.